src/App.tsx

```diff
@@ -1,9 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type State = string;
+
+type Movies = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string
+};
+
+export function getFilteredMovies(movies: Movies[], filter: string) {
+  const visibleMovies = [...movies];
+
+  return visibleMovies.filter(movie => movie.title
+    .toLowerCase()
+    .includes(filter.toLowerCase().trim())
+    || movie.description.toLowerCase().includes(filter.toLowerCase().trim()));
+}
```

Comment:
filter is not mutable method
```suggestion
export function getFilteredMovies(movies: Movies[], filter: string) {
  return movies.filter(movie => movie.title
    .toLowerCase()
    .includes(filter.toLowerCase().trim())
    || movie.description.toLowerCase().includes(filter.toLowerCase().trim()));
}
```

Reply:
okay